javascript/401/Challenge-13/module/brackets.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+class Brackets {
+  constructor() {
+    this.front = null;
+    this.back = null;
+  }
+  validateBrackets(str) {
+    let count = 0;
+    let queue = [];
+    let map = {
+      '(': ')',
+      '[': ']',
+      '{': '}',
+    };
+    let lib = ['(', ')', '[', ']', '{', '}'];
+
```

Comment:
This works great! I think you are using a queue as though it was a stack. A stack would be the preferred data structure to use for this challenge.